draft-ietf-quic-qpack.md

```diff
@@ -1128,7 +1141,7 @@ The entries in the following table are registered by this document.
 
 ## Stream Type Registration
 
-This document creates two new settings in the "HTTP/3 Stream Type" registry
+This document creates two new streams in the "HTTP/3 Stream Type" registry
```

Comment:
Perhaps "types" or "entries"?

Reply:
The change addressed a bug in the spec. I'm happy to use entries but would do so consistently through the document (settings, errors, stream)

Reply:
Well, arguably, the other two actually are creating new settings and error types.  But yes, if you want to fix the bug and remain consistent, this should probably be "types".